tests/test_util_data_schema.py

```diff
@@ -0,0 +1,109 @@
+"""Test util.data_schema module."""
+
+import unittest
+import json
+from unittest.mock import MagicMock
+from aind_data_access_api.util.data_schema import get_quality_control
+from aind_data_schema.core.quality_control import QualityControl
+
+
+class TestUtilDataSchema(unittest.TestCase):
+    """Test methods in data schema."""
+
+    def test_get_qc_id(self):
+        """Test get_quality_control function."""
+        # Get json dict from test file
+        with open("./tests/resources/helpers/quality_control.json", "r") as f:
```

Comment:
Instead of loading files individually in the unit tests, it's usually better to use the `setUpClass` classmethod. Take a look at [test_utils.py](https://github.com/AllenNeuralDynamics/aind-data-access-api/blob/2ff41d5bdf15859247534b7e307bad413955a85f/tests/test_utils.py#L27)

Reply:
@jtyoung84 mentioned the paths may cause issues across different OS. @jtyoung84 could you check if this works on your Linux machine?

Reply:
The tests on github run on linux, I updated it to use pathlib

Reply:
Paths can have issues on different machines and different envs. I found that something like this is a reliable way to define them in tests:
https://github.com/AllenNeuralDynamics/aind-data-access-api/blob/dev/tests/test_utils.py